setup.py

```diff
@@ -7,7 +7,7 @@
 
 MAJOR = 0
 MINOR = 2
-MICRO = 1
+MICRO = 2
 
 IS_RELEASED = False
```

Comment:
oh, I think I see what you mean. With Simon, we figured it was very easy to forget to switch back `IS_RELEASED` to False, so we do the flip back and forth in one PR